core/dbt/events/types.py

```diff
@@ -863,7 +829,7 @@ class InvalidVarsYAML(ErrorLevel, Cli, File):
     code: str = "A008"
 
     def message(self) -> str:
-        return "The YAML provided in the --vars argument is not valid.\n"
+        return "The YAML provided in the --vars argument is not valid."
 
 
 # TODO: Remove? (appears to be uncalled)
```

Comment:
Can we remove these too?

Reply:
This highlighted wrong.  I'm referring to `HandleInternalException`, `CatchRunException`, `DetailsHandleGenericException`

Reply:
yep can do!

Reply:
also removing `MessageHandleGenericException`, since it doesn't have a code and appears to be uncalled